akka-remote/src/main/scala/akka/remote/serialization/DaemonMsgCreateSerializer.scala

```diff
@@ -41,14 +41,31 @@ private[akka] final class DaemonMsgCreateSerializer(val system: ExtendedActorSys
 
       def deployProto(d: Deploy): DeployData = {
         val builder = DeployData.newBuilder.setPath(d.path)
-        if (d.config != ConfigFactory.empty)
-          builder.setConfig(oldSerialize(d.config))
-        if (d.routerConfig != NoRouter)
-          builder.setRouterConfig(oldSerialize(d.routerConfig))
-        if (d.scope != NoScopeGiven)
-          builder.setScope(oldSerialize(d.scope))
-        if (d.dispatcher != NoDispatcherGiven)
+
+        {
```

Comment:
yeah, was an if-else before, wanted to keep it a block/scope but didn't need the if anymore.